src/dotnet-fssrgen/project.json

```diff
@@ -1,51 +0,0 @@
-{
-    "title": "F# String Response Generator - dotnet cli tool",    
-    "version": "3.3.0",
-    "authors": ["Don Syme","Enrico Sada","Jared Hester"],
```

Comment:
you need to move these too, they'll be added to the `.nuspec` generated by `dotnet pack`